pallets/parachain-staking/src/lib.rs

```diff
@@ -1803,6 +1917,21 @@ pub mod pallet {
 			Ok(())
 		}
 
+		fn delegator_is_kicked(delegation: &StakeOf<T>, collator: &T::AccountId) -> DispatchResult {
```

Comment:
It would be clearer to name it something like `kick_delegator`.

Reply:
fixed in b03f3c6